### PR TITLE
Fix Katex overflow

### PR DESCRIPTION
**Pull request by danisztls**
Issue #294

### CODE REVIEW

**Comment by sylhare** [comment]
That could be a solution; but for big equations there might be some superposition happening.
I would like sot see how it would react in this particular case.

**Comment by danisztls** [comment]
> That could be a solution; but for big equations there might be some superposition happening.
> I would like sot see how it would react in this particular case.

horizontally?

**Comment by sylhare** [comment]
yes, a big vertical one like:

text

equation
---------
equation

text

**Comment by danisztls** [comment]
On Y there is no problem. But on X really long lines would be clipped because wrapping was disabled, which I fixed on last commit.

File: _sass/includes/_katex.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.katex-display {
+  overflow: visible;
+}
+
+.katex {
+  whitespace: wrap;
```
Comment (sylhare) [inline _sass/includes/_katex.scss:6]: `whitespace` doesn't seem to be a correct attribute, I see `white-space` online but no `wrap` associated to it.

Comment (sylhare) [reply _sass/includes/_katex.scss:6]: It does not seem to be doing anything

**Comment by sylhare** [comment]
Which Katex sample did you use to test the CSS? Maybe we can add a super long one in the article for reference :)

**Comment by danisztls** [comment]
I just copied and pasted a number of times the sample at https://sylhare.github.io/Type-on-Strap/2014/11/28/markdown-and-html.html 😁

**Comment by sylhare** [comment]
Yeah from what I see, I would rather keep the scroll bar than having an equation going overboard.
Overflowing may happen more often with mobile display too.

Maybe it's more of an issue to open with KaTex?

**Pull request closed without merging**